content/docs/guides/converting-wordpress-to-tina.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Migrating WordPress Content
+last_edited: '2024-11-04T04:31:04.917Z'
+next: ''
+previous: ''
+---
+
+## Introduction
+
+This guide is a walkthrough using a **WordPress to Markdown converter**, which is recommended for most use cases where you want to migrate your content from WordPress to Tina.
+
+Other conversion options that are not covered in this guide include:
+
+- Using the WordPress Rest API - best for extracting the pure html for page or post content in JSON format
+- Using the WPGraphQL plugin - best for cases where you want to query and extract specific data and content
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) is installed
+- You have set up a [Tina starter](https://tina.io/docs/introduction/using-starter)
+
+## Getting Started
+
+### Converting WordPress to Markdown
+
+1\. Export your Wordpress content by going to [https://wordpress.com/export/](https://wordpress.com/export/) and select **Export All**
+
+> 💡 **Note:** Name it **export.xml** to make it easier for you in the next step
+
+Use [wordpress-export-to-markdown](https://github.com/lonekorean/wordpress-export-to-markdown) to convert your WordPress content into Markdown.
+
+2\. In the folder where you've saved your exported file **export.xml**, run:
+
+```powershell
+npx wordpress-export-to-markdown --post-folders=false --include-other-types=true
+```
+
+3\. When prompted, make sure the filename is correct, and use the default values for the other prompts.
+
+This creates an `output` folder that groups your posts and pages into separate folders.
+
+### Import data to TinaCMS
+
+#### Migrating images to TinaCloud
+
+1\. In the output folder, drag the image folders `output/post/images` and `output/page/images` to your Tina app's `public/uploads` folder.
+
+> 💡 **Note 1:** You can rename the folder before dragging to prevent duplicate image names being replaced
+
+> 💡 **Note 2:** You can also set up alternative hosting with your preferred media manager.
+
+#### Migrating Posts
+
+1\. Navigate to your output's posts folder `output/post`, and run:
+
+```powershell
+ren * *.mdx
+```
+
+Which will convert your files from Markdown to MDX.
+
+2\. Drag all the MDX files in `output/post` to your Tina app's `content/posts` folder, now your posts have been migrated to Tina!
+
+#### Migrating Pages
+
+<WarningCallout body="If you are having trouble with this step, feel free to contact a developer for assistance in the chat, or through one of our community channels." />
+
+1\. Create a new folder in your Tina app's `content` folder named `wp-pages`.
+
+2\. Create a new file in your Tina app's `tina/collection` folder named `wp-page.ts` and use the template below, you can change this later to fit your page's schemas better.
```

Comment:
This is a big reduction – what kind of data do we get in these WP markdown files specifically? Should they set up block based editing? How can we look at the converted data and turn it into a schema?

Do we want a seperate guide that goes over general migration tips (ex. this is the markdown we have, how can we create a schema that fits it) – then we could just link that and use the wordpress markdown as the example

**Counterargument**
This guide looks really straightforward as is – which is great.

Reply:
As per my conversation with @Marxoz, I've created a new PBI to create a general migration guide, see #2438.